cmd/podman/varlink/io.podman.varlink

```diff
@@ -386,9 +376,15 @@ type Runlabel(
     opts: [string]string
 )
 
-# GetVersion returns a Version structure describing the libpod setup on their
-# system.
-method GetVersion() -> (version: Version)
```

Comment:
please add API documentation like many of the other functions. thanks.

Reply:
Added it back.